### PR TITLE
Fixed links to interview.

### PR DESCRIPTION
**Pull request by patcon**
Resolves bug#3 in https://github.com/edgi-govdata-archiving/100days/issues/53

### CODE REVIEW

File: pursuing-toxic-agenda.html [hunk 1]
```diff
@@ -581,7 +581,7 @@ <h2 id="h.va0wuax2pbgz">Environmental Data Justice</h2>
...
+            <p><a href="#ftnt_ref25" id="ftnt25">[25]</a> Lindsey Dillon, <a href="/mustafa-ali-interview/">interview with Mustafa </a><a href="#id.qskbsdioqlsb">Ali</a>, August 8, 2017. For example, NEJAC was instrumental in pushing the EPA brownfields program to incorporate economic equity considerations, such as the Brownfields Job Training Program. Many brownfields (or industrial properties requiring toxic cleanup) are located in poor communities that struggle with unemployment, and are at risk of being displaced by gentrification that often follows from environmental cleanup and greening projects. The Brownfield Job Training Program is an attempt to bring some of the benefits of environmental cleanup to the communities that have been most impacted by toxic waste. </p>
```
Comment (suchthis) [inline pursuing-toxic-agenda.html]: Can we change this to:

`<a href="/mustafa-ali-interview/">interview with Mustafa Ali</a>`

File: pursuing-toxic-agenda.html [hunk 2]
```diff
@@ -868,7 +868,7 @@ <h2 id="h.va0wuax2pbgz">Environmental Data Justice</h2>
...
         </div>
         <div>
-            <p><a href="#ftnt_ref119" id="ftnt119">[119]</a> Dillon, <a href="#id.qskbsdioqlsb">interview with Mustafa </a><a href="#id.qskbsdioqlsb">Ali</a>, August 8, 2017; also see Tony Lo Presti,&rdquo; Realizing the Promise of Environmental Civil Rights: The Renewed Effort to Enforce Title VI of the Civil Rights Act of 1964,&rdquo; Administrative Law Review 59, (June 28, 2013): <a href="https://ssrn.com/abstract%3D2286529">https://ssrn.com/abstract=2286529</a>.</p>
+            <p><a href="#ftnt_ref119" id="ftnt119">[119]</a> Dillon, <a href="/mustafa-ali-interview/">interview with Mustafa </a><a href="#id.qskbsdioqlsb">Ali</a>, August 8, 2017; also see Tony Lo Presti,&rdquo; Realizing the Promise of Environmental Civil Rights: The Renewed Effort to Enforce Title VI of the Civil Rights Act of 1964,&rdquo; Administrative Law Review 59, (June 28, 2013): <a href="https://ssrn.com/abstract%3D2286529">https://ssrn.com/abstract=2286529</a>.</p>
```
Comment (suchthis) [inline pursuing-toxic-agenda.html]: Can we change this to:

`<a href="/mustafa-ali-interview/">interview with Mustafa Ali</a>`

**Review by suchthis — COMMENTED** [review]
Thanks so much! Left two inline comments and can we add a link to interview from line 64? That fixes the second part of one of the issues in #53.

**Review by dcwalk — APPROVED** [review]
Looks great!

**Pull request merged**